### PR TITLE
DB-3595: [next-wordpress-starter] Change to use SSR by Default

### PR DESCRIPTION
**Pull request by backlineint**
## What changes were made?

Changed next-wordpress-starter to use SSR by default. Created ISR example for reference.

## How have the changes been tested?

Tested manually locally in both dev and prod mode.

## Additional information

Don't forget to [add a changeset](https://github.com/pantheon-systems/decoupled-kit-js#generating-a-changeset) if needed!

### CODE REVIEW

File: starters/next-wordpress-starter/pages/examples/index.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+import PageHeader from "../../components/page-header";
+import Layout from "../../components/layout";
+import { withGrid, PostGridItem } from "../../components/grid";
+
+import { getFooterMenu } from "../../lib/Menus";
+import { getLatestPosts } from "../../lib/Posts";
+
+export default function PostsListTemplate({ menuItems, SSRPosts }) {
+  const PostGrid = withGrid(PostGridItem);
```
Comment (CobyPear) [inline starters/next-wordpress-starter/pages/examples/index.jsx:12]: Looks like this isn't being used

**Review by CobyPear — CHANGES_REQUESTED** [review]
LGTM, just one lingering unused component on the examples index page. 

I am unable to get an authenticated request to work on the auth-api page. Probably not a regression, but I'm wondering if you are experiencing the same?

File: starters/next-wordpress-starter/pages/examples/index.jsx (hunk 1 above)
Comment (backlineint) [reply starters/next-wordpress-starter/pages/examples/index.jsx]: Cleaned up the unused stuff here.

**Comment by backlineint** [comment]
> I am unable to get an authenticated request to work on the auth-api page. Probably not a regression, but I'm wondering if you are experiencing the same?

Tried this out and it also wasn't working for me. Not my fault, I swear :) We'll have to create a bug ticket for it.

**Review by CobyPear — APPROVED** [review]

**Comment by CobyPear** [comment]
> > I am unable to get an authenticated request to work on the auth-api page. Probably not a regression, but I'm wondering if you are experiencing the same?
> 
> Tried this out and it also wasn't working for me. Not my fault, I swear :) We'll have to create a bug ticket for it.

Ah dang. Probably for the best, apparently `btoa` is deprecated and we shouldn't have used it in the first place!

**Pull request merged**